CONTRIBUTING.md

```diff
@@ -45,3 +45,19 @@ This list is not comprehensive, meaning if you want to include more detail than
 ## Go style
 
 This will be checked automatically by our CI linter bot.
+
+## Releasing
+
+In order to create a new release:
+* Update the VERSION file:
+  echo "x.y.z" > VERSION
+* Commit with the message "Bump version: x.y.z", being x.y.z the version. Please use semantic versioning.
+  git commit -m "Bump version: x.y.z" -s
+* Tag with vx.y.z
+  git tag vx.y.z
+* Push
+  git push --tags origin master
+* Create the changelog:
```

Comment:
we should add an extra tag for 'devel', 'alpha', and 'beta' so when we have a numbered only release it is implied to be a stable production release.